libs/handoversidesheet/src/lib/ui-sidesheet/DetailsTabs.tsx

```diff
@@ -71,6 +71,10 @@ export const DetailsTab = ({ commpkg }: DetailsTabProps) => {
             <td> Mc Packages Missing PunchOut</td>
             <td>{stringCell(`${commpkg.remainingPunchOutCount || 0}`)}</td>
           </tr>
+          <tr>
+            <td> Mc Packages Missing PunchOut</td>

```

Comment:
Correct title?

Reply:
Was a little fast when commiting, it has been fixed